src/pages/projects/AddProjectContributorForm.tsx

```diff
@@ -0,0 +1,107 @@
+import { Box, Button, MenuItem, TextField } from '@mui/material';
+import { useFormikContext } from 'formik';
+import { useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useDispatch } from 'react-redux';
+import { ContributorSearchField } from '../../components/ContributorSearchField';
+import { StyledRightAlignedFooter } from '../../components/styled/Wrappers';
+import { setNotification } from '../../redux/notificationSlice';
+import {
+  CristinProject,
+  ProjectContributor,
+  ProjectContributorType,
+  ProjectFieldName,
+} from '../../types/project.types';
+import { CristinPerson } from '../../types/user.types';
+import { dataTestId } from '../../utils/dataTestIds';
+
+const roles: ProjectContributorType[] = ['ProjectManager', 'ProjectParticipant'];
+
+interface AddProjectContributorFormProps {
+  hasProjectManager: boolean;
+  toggleModal: () => void;
+}
+
+export const AddProjectContributorForm = ({ hasProjectManager, toggleModal }: AddProjectContributorFormProps) => {
+  const { t } = useTranslation();
+  const dispatch = useDispatch();
+  const { values, setFieldValue } = useFormikContext<CristinProject>();
+  const [searchTerm, setSearchTerm] = useState('');
+  const [selectedContributorRole, setSelectedContributorRole] = useState(hasProjectManager ? roles[1] : roles[0]);
```

Comment:
Syns det er enklere om man kan se verdien direkte, uten å måtte se i lista den viser til

```suggestion
  const [selectedContributorRole, setSelectedContributorRole] = useState<ProjectContributorType>(
    hasProjectManager ? 'ProjectParticipant' : 'ProjectManager'
  );
```